### PR TITLE
add quantcast ID submodule

### PR DESCRIPTION
**Pull request by mckurt**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change
This adds quantcast user ID submodule. The submodule reads Quantcast first party cookie and passes the value to prebid bidding adapters. Currently, the submodule doesn't need any configuration parameters. The code is tested using with unit tests and userId_example.html. The code coverage on the submodule is 85.71%. Please contact asig@quantcast.com for any issues.

Link to the documentation PR: https://github.com/prebid/prebid.github.io/pull/2339

### CODE REVIEW

File: integrationExamples/gpt/userId_example.html [hunk 1]
```diff
@@ -83,9 +83,10 @@
         var adUnits = [
             {
                 code: 'test-div',
-                sizes: [[300,250],[300,600],[728,90]],
                 mediaTypes: {
-                    banner: {}
+                    banner: {
+                        sizes: [[300,250],[300,600],[728,90]]
```
Comment (mckurt) [inline integrationExamples/gpt/userId_example.html:88]: I noticed that a recent PR tackled the same issue (https://github.com/prebid/Prebid.js/pull/5606), but I was still getting errors if the sizes aren't defined inside the _banner_ object. I can move this to a different PR if that'd be better.

File: test/spec/modules/quantcastIdSystem_spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { quantcastIdSubmodule, storage } from 'modules/quantcastIdSystem.js';
+
+describe('QuantcastId module', function () {
+  beforeEach(function() {
+    window.document.cookie = '__qca= ; expires = Thu, 01 Jan 1970 00:00:00 GMT;';
```
Comment (smenzer) [inline test/spec/modules/quantcastIdSystem_spec.js:5]: can you use the `storage.setCookie()` function for this instead?

**Review by smenzer — CHANGES_REQUESTED** [review]
generally looks good to me, just a minor change in the test around cookies.

**Review by jlukas79 — COMMENTED** [review]
Looks good to me, will approve once changes above completed and conflicts resolved

**Comment by mckurt** [comment]
@jlukas79 @smenzer Thanks for the review. I addressed the comment. Will attach a link to the documentation PR soon.

**Comment by mckurt** [comment]
Here is the documentation PR: https://github.com/prebid/prebid.github.io/pull/2339 

I also updated the description.

**Review by jlukas79 — APPROVED** [review]

**Review by smenzer — APPROVED** [review]
LGTM

**Pull request merged**